homeassistant/components/solarlog/strings.json

```diff
@@ -10,6 +10,7 @@
         },
         "data_description": {
           "host": "The hostname or IP address of your Solar-Log device.",
+          "name": "The prefix to be used for your Solar-Log sensors",
```

Comment:
Instead of adding this one, I think it might be better to remove it.

The names shouldn't be asked in flows. Adding it here now and removing it later will just add an unneeded translated string to our translating community.

../Frenck

Reply:
From a formal perspective, as the removal of the name would need an entry migration, I would do this in a separate PR, agreed?